.github/workflows/TestJuliaVersions.yml

```diff
@@ -28,16 +28,16 @@ jobs:
         id: cache-libprecice
         uses: actions/cache@v2
         with:
-          path: libprecice3_3.1.0_focal.deb
-          key: libprecice3_3.1.0_focal.deb1
-          restore-keys: libprecice3_3.1.0_focal.deb1
+          path: libprecice3_3.2.0_noble.deb
```

Comment:
Why is the package version here hard-coded, whereas above we run on `[ubuntu-latest]` ?

I would also advice to update the action versions used herein (e.g., `actions/checkout@v2` is I think rather old). You can just enable the dependabot in the repo settings to take care of this automatically.